lib/components/embed.js

```diff
@@ -22,18 +22,63 @@ function renderYoutube (props) {
     width={480} height={270}></amp-youtube>);
 }
 
-function render ({ props }) {
-  const { embedType } = props;
-  let embed = '';
+function renderFacebook (props) {
+  const { url, embedAs } = props;
 
-  if (embedType === 'image') {
-    embed = renderImage(props);
+  if (embedAs === 'post') {
+    return (<amp-facebook width={486} height={657}
+      layout='responsive' data-embed-as='post'
+      data-href={url}></amp-facebook>);
   }
 
-  if (embedType === 'youtube') {
-    embed = renderYoutube(props);
+  if (embedAs === 'video') {
+    return (<amp-facebook width={480} height={270}
+      layout='responsive' data-embed-as='video'
+      data-href={url}></amp-facebook>);
   }
 
+  return '';
+}
+
+function renderTwitter (props) {
+  const { id } = props;
+
+  return (<amp-twitter width={486} height={657}
```

Comment:
I'm using proposed values from the AMP documentation, it's responsive so it only needs the aspect ratio.